src/main/java/handlers/ExceptionHandler.java

```diff
@@ -18,13 +22,18 @@
     public Response ResponseBuilder(Exception e) {
         ErrorMessage errorMessage;
         if (e instanceof TwitterException) {
+            logger.debug("Error recognized as TwitterException");
```

Comment:
whenever you are logging in error path with exceptions it's useful to log the exception as well.

Reply:
This error is logged in when the exception is first thrown, in the class that is calling this handler